Gemfile

```diff
@@ -15,15 +15,11 @@ gem 'openssl'
 gem 'figaro', '>= 1.1.1'
 
 gem 'decidim', DECIDIM_VERSION
-gem 'decidim-verifications-salou_census', git: "git@gitlab.coditdev.net:decidim/decidim-verifications-salou_census.git", tag: "v0.3.1"
+gem 'decidim-verifications-salou_census', path: '../'
```

Comment:
I think you commited this modification from development.

Reply:
Yes. I'm waiting for [this](https://gitlab.coditdev.net/decidim/decidim-verifications-salou_census/merge_requests/3) to be merged to update de Gemfile

Reply:
@tramuntanal can you resolve this, please? I can't merge